PublicationsRouter/v2/sword-out/REPOSITORY.md

```diff
@@ -4,19 +4,44 @@ This document provides information on how to get your repository set up to take
+First you will need a Router account, and set up the SWORDv2 credentials in the 'Manage connection settings' section on your account.  You need to provide the following:
+
+* The repository URL
+* Select a Repository Configuration ([Eprints Native](#eprints), [Eprints RIOXX](#eprints-rioxx-plugin), [DSpace native](#dspaceother))
+* Select an Eprints deposit location if required (Manage Deposit, Review Queue)
+* A username and password for a user in your repository who has the rights to create content via SWORDv2.
+   * For Eprints deposits into *Manage Deposits* a User role is required.
+   * For Eprints deposits into *Review* queue an Editor or Admin role is required.
+   * For DSpace deposits into a collection an Admin role on the collection is required.
+* A collection URL into which the system will deposit the content
+   * Eprints collection: http://eprints.domain.ac.uk/id/contents
+   * DSpace collection: `http://dspace.domain.ac.uk/swordv2/collection/<handle-id>/<collection-id>` (usually obtained from the collection URL `http://dspace.domain.ac.uk/handle/<handle-id>/<collection-id>` find out more: [DSpace URLs and Identifiers](https://wiki.duraspace.org/display/DSDOC5x/Functional+Overview#FunctionalOverview-PersistentURLsandIdentifiers))
+* Your preferred packaging format for zip files being deposited. [more info](https://github.com/sherpaservices/Public-Documentation/blob/master/PublicationsRouter/v2/api/Packaging.md#what-package-formats-are-supported)
+   * http://purl.org/net/sword/package/SimpleZip (by default)
+   * https://pubrouter.jisc.ac.uk/FilesAndJATS
 
 ## EPrints
 
-Router is only certified to work with EPrints 3.3.
+Router is only certified to work with EPrints 3.3. There are 2 different formats to deposit notifications into Eprints: Vanilla Eprints and Eprints RIOXX. 
+
+### Vanilla Eprints
+
+Vanilla EPrints is the native format and it is configured by default to accept incoming requests via [SWORDv2](https://wiki.eprints.org/w/SWORD_2.0), so the router's deposit mechanism will automatically work.  
+
+### Eprints RIOXX Plugin
+
+Eprints RIOXX format provides consistency to the metadata and is designed to support the consistent tracking of open-access research publications across scholarly systems.
+
+The final version of the [*Jisc PubRouter RIOXXplus Connector Plugin*](http://wiki.eprints.org/w/Jisc_Publications_Router) is now available and it is necessary to install it on your Eprints repository in order to receive notifications in this format.
+
+In summary, the following would need to be done:
+
+1.	Install the Jisc PubRouter RIOXXplus Connector Plugin from the Eprints Bazaar (this would normally be done by an Eprints administrator)
+
+2.	Configure the plugin - it requires your PubRouter API key to be entered (again a job for an Eprints administrator)
+
+3.	Update your PubRouter account to  change the repository configuration from Eprints Native to Eprints RIOXXplus.
```

Comment:
Add a sentence after the bullets:

`For further information see the Eprints RIOXXplus plugin [help page](http://wiki.eprints.org/w/Jisc_Publications_Router).`